contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1205,10 +1116,8 @@ export const ContentNode = new Resource({
         .then(data => {
           return db[CHANGES_TABLE].put({
             key: id,
-            mods: {
-              target,
-              position,
-            },
+            target,
```

Comment:
As part of move, we'll also need to make sure original parent has `changed=True` set, otherwise users won't be able to publish their channels if they delete a node

Reply:
This happens in the backend but we need to propagate to the frontend too, yeah.